constantine.nimble

```diff
@@ -607,6 +607,9 @@ const testDesc: seq[tuple[path: string, useGMP: bool]] = @[
   ("tests/t_ethereum_verkle_primitives.nim", false),
   ("tests/t_ethereum_verkle_ipa_primitives.nim", false),
 
+  # Signatures
+  ("tests/ecdsa/t_ecdsa_verify_openssl.nim", false),
```

Comment:
This can be commented out until Ubuntu 25.04 is available in CI (probably June)

Reply:
Are the non LTS releases ever part of the Github Actions CI images?